src/lib/components/ToggleGroup/root.svelte

```diff
@@ -49,6 +50,7 @@
 		readonly loop: boolean;
 		readonly rovingFocus: boolean;
 		readonly disabled: ToggleGroupRootProps['disabled'];
+		readonly role: ToggleGroupRootProps['role'];
```

Comment:
Is this needed here? Since you're not using it on any other component

Reply:
It is needed for type inference when setting content. In terms of setting the context in the first place, technically, it isn't required, but I include it to keep it consistent with other props.

Reply:
What I mean is, you're not using rootContext.role anywhere, you're only using the role that comes from `export let role`, so this doesn't end being used, right?

Reply:
I see. that Is correct; there is no need to subscribe to changes.